### PR TITLE
Prepare for intent to prototype

### PR DESCRIPTION
**Pull request by jbingham**
It's been over a year since updating the Explainer, and some of the concerns around MLIR and the level of abstraction have already been addressed in the Community Group.

### CODE REVIEW

**Review by anssiko — APPROVED** [review]
Thanks @jbingham! Here's minor patch to sync the spec with this update, PTAL: https://github.com/webmachinelearning/model-loader/pull/8

**Comment by yuhonglin** [comment]
It seems the usage of this API (to align with WebNN API) has not been updated. Are we going to do that in a separate pr?

**Comment by anssiko** [comment]
I assume you refer to the sample code? Given this is an explainer doc, we can iterate on the doc (and spec draft too) flexibly to align with the prototyping work. So yes, more PRs welcome.

(Also, welcome to the group @yuhonglin!)

**Comment by yuhonglin** [comment]
Thanks, Anssi! I understand that there is no need to have the explainer freezed for the moment. But we plan to make some significant changes,
1. We want to make the API to align with WebNN, e..g. sharing the interface MLContext etc.
2. We want the API be able to support multiple model formats by adding extra arguments. This is important even for the moment to pass the API preview.

We have a doc discussing this. May I have your email address? Then I can share the doc with you.

**Comment by jbingham** [comment]
It should be ok if the final spec and implementation is a little different. It's sample code or pseudo-code to give an idea of how the API might work. 

I'll merge and we can update if needed. Thanks!

File: explainer.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ var options = {
 const modelLoader = navigator.ml.createModelLoader();
 const model = await modelLoader.load(modelUrl)
 const compiledModel = await model.compile()
```
Comment (huningxin) [inline explainer.md:40]: Thanks for changing the `predict` to `compute` as we are aligning the Model Loader and WebNN. Probably the `compile` step can folded into `load` step. However I am fine of it at the current stage.

**Review by huningxin — COMMENTED** [review]
Thanks @jbingham . It looks good to me.

**Comment by anssiko** [comment]
>Thanks, Anssi! I understand that there is no need to have the explainer freezed for the moment. But we plan to make some significant changes,

@yuhonglin, your understanding is correct. At the explainer phase, the closer the explainer tracks the prototyping work, the better. Feel free to update the explainer at any stage. You just got mail so you have my address now :)

**Pull request merged**